src/main/java/tech/jhipster/lite/generator/server/springboot/mvc/web/domain/SpringBootMvcsModulesFactory.java

```diff
@@ -23,6 +23,10 @@ public class SpringBootMvcsModulesFactory {
   private static final JHipsterSource MAIN_SOURCE = SOURCE.append("main");
   private static final JHipsterSource TEST_SOURCE = SOURCE.append("test");
 
+  private static final JHipsterSource JACKSON_MAIN_SOURCE = from("server/springboot/jackson/main");
+  private static final JHipsterSource JACKSON_TEST_SOURCE = from("server/springboot/jackson/test");
+  private static final String WIRE_JACKSON_CONFIG = "wire/jackson/infrastructure/config";
```

Comment:
Subfolder of `infrastructure` folder are either `primary` or `secondary`:
```suggestion
  private static final String WIRE_JACKSON_CONFIG = "wire/jackson/infrastructure/primary";
```

Reply:
![image](https://github.com/user-attachments/assets/025a83f4-3381-45aa-92fb-4d8238cccecd)

Reply:
You need the update the JacksonConfiguration.java content too:
`package {{packageName}}.wire.jackson.infrastructure.config;` -> `package {{packageName}}.wire.jackson.infrastructure.primary;`